backend/typescript/middlewares/validators/campValidators.ts

```diff
@@ -16,10 +16,15 @@ import {
 } from "./util";
 import { getErrorMessage } from "../../utilities/errorUtils";
 
-const createCampSessionDTOValidator = (campSession: {
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  [key: string]: any;
-}): boolean => {
+enum CreateOrUpdate {
+  CREATE = 1,
+  UPDATE = 2,
+}
+
+const createUpdateCampSessionDTOValidator = (
```

Comment:
`createUpdateCampValidator` calls `createUpdateCampSessionDTOValidator`. This caused the initial problem of "campers must be empty", this validation doesn't make sense when viewing/updating a camp, only when creating.